unified-prototype/testbed-android-application/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
❓ Shouldn't this live in a `main/resources` dir and not directly under `main`?

Reply:
Apparently not. When I move it I get:
```
Execution failed for task ':testbed-android-application:createDebugCompatibleScreenManifests'.
> Error while evaluating property 'variantOutputs.$0.versionCode' of task ':testbed-android-application:createDebugCompatibleScreenManifests'.
   > Manifest file does not exist: /Users/jvandort/work/declarative-prototypes/unified-prototype/testbed-android-application/src/main/AndroidManifest.xml
```